modules/editorial-metadata/rest/editorial-metadata-endpoint.php

```diff
@@ -6,15 +6,15 @@
 
 namespace VIPWorkflow\Modules\EditorialMetadata\REST;
 
-use VIPWorkflow\Modules\Editorial_Metadata;
+use VIPWorkflow\Modules\EditorialMetadata;
 use VIPWorkflow\VIP_Workflow;
 use WP_Error;
 use WP_REST_Request;
 use WP_Term;
 
 defined( 'ABSPATH' ) || exit;
 
-class EditEditorialMetadata {
+class EditorialMetadataEndpoint {
```

Comment:
REST endpoints have been standardized to end with `Endpoint`. I just like this because it's easier to tell the difference between filenames in the editor and gives REST files an easy pattern.